src/MainWindow.cpp

```diff
@@ -1406,3 +1397,11 @@ void MainWindow::aboutQt()
 {
     QMessageBox::aboutQt(this);
 }
+
+
+QMessageBox::StandardButton MainWindow::openNewFileQuestionDialog(QString oldFilepath) {
```

Comment:
code style - newline before scope opening brace
pass by value - the `QString oldFilepath`